### DISCUSSION TITLE
Candles handling types

### DISCUSSION THREAD

**Discussion by jasonmellone** [KmiQ/coinbase-advanced-python#46] [Ideas]
Would you be open to candles handling types? Eg start behaving as a datetime, ohlc being cast to float...etc. Happy to contribute but before doing anything thought to ask

**Comment by KmiQ** [comment]
> Would you be open to candles handling types? Eg start behaving as a datetime, ohlc being cast to float...etc. Happy to contribute but before doing anything thought to ask

Sure thing! How do you feel about opening a PR?
